src/components/TrailList.js

```diff
@@ -85,9 +94,9 @@ class TrailList extends React.Component {
             <TableBody style={{overflowY: 'auto', height: '100%', width: '95%'}}>
               {orphanCount ? (
                 <TableRow
-                  className={selected === null ? 'selected' : ''}
+                  className={ orphanRowSelected ? 'selected': ''}
```

Comment:
It seems possible for both orphanRowSelected to be true and for a trail to be selected at the same time. 

The hydrant reducer should take care of deselecting the features when a new trail is selected, but would this row stay highlighted until it was manually deselected? Could add a check like `orphanRowSelected && !selectedTrail ? ...`